daq/host.py

```diff
@@ -658,7 +661,7 @@ def clear_port(gcp_instance, port):
         }
         gcp_instance.publish_message('daq_runner', 'test_result', result)
 
-    def _record_result(self, name, run_info=True, current=None, **kwargs):
+    def _gcp_record_result(self, name, run_info=True, current=None, **kwargs):
```

Comment:
why is this now specific to gcp?  I think it's bleeding the abstraction... this would be the inflection point for any non-gcp recording as well...